.markdownlint.json

```diff
@@ -1,4 +1,12 @@
+// /.markdownlint.json -*-config-*-
```

Comment:
Looks like github thinks that json comments are an error. 
That's annoying but getting all this red is also?

Reply:
Converted to `markdownlint.yml`, where comments are OK to be displayed. Also, pulled config from https://github.com/DavidAnson/markdownlint/blob/main/schema/.markdownlint.yaml. I'll tweak it.